quantum/rgblight_breathe_table.h

```diff
@@ -1,468 +1,117 @@
-#ifndef RGBLIGHT_EFFECT_BREATHE_TABLE
+#pragma once
+
```

Comment:
All source files except rgblight.c will not include table.h. So there is no need for include guards or '#pragma once' in practice.

```suggestion
```

Reply:
In practice, yes, but this is a good safety net just in case.